docs/getting-started/concepts.md

```diff
@@ -3,19 +3,33 @@ template: main.html
 ---
 
 # Iter8
-Iter8 is a metrics-driven release optimizer built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well, are secure, and maximize business value.
+Iter8 is the Kubernetes testing platform built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well and maximize business value.
```

Comment:
> Iter8 is the Kubernetes testing platform built for DevSecOps, MLOps, SRE and data science teams.

Do we want to say "**the** Kubernetes testing platform" or "**a** Kubernetes testing platform"?

Reply:
I prefer **the** ; **the one and only** is really I want to see but that is a bit too cheesy :)